testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -69,3 +79,19 @@ def test_optimize_cutoff(universe, lipid_heads):
     cutoff, N = optimize_cutoff(universe, lipid_heads, pbc=True)
     assert N == 2
     assert_almost_equal(cutoff, 10.5, decimal=4)
+
+def test_pdc_on_off(universe, lipid_heads):
+    lfls_pbc_on = LeafletFinder(universe, lipid_heads, pbc=True)
+    lfls_pbc_off = LeafletFinder(universe, lipid_heads, pbc=False)
+    assert lfls_pbc_on.graph.size() > lfls_pbc_off.graph.size()
+
+def test_cutoff_update(universe, lipid_heads):
+    lfls_ag = LeafletFinder(universe, lipid_heads, cutoff = 15.0, pbc=True)
+    lfls_ag.update(cutoff=1.0)
+    assert_almost_equal(lfls_ag.cutoff, 1.0, decimal=4)
+
+def test_write_selection(universe, lipid_heads, tmpdir):
+    lfls_ag = LeafletFinder(universe, lipid_heads, cutoff = 15.0, pbc=True)
+    with tmpdir.as_cwd():
+         filename = lfls_ag.write_selection('leaflet.vmd')
+         assert len(open('leaflet.vmd').readlines()) == 50, "Leaflet write selection not working"
```

Comment:
Could you also test the inside of the file, i.e. say that the selection for say a vmd file would work here?